README.md

```diff
@@ -3,104 +3,93 @@ Common instructions to pull and setup a Symfony/MySql porject
 
 ## Local setup
 
-1) Setup PHP >= 8.1 and extensions
+Setup PHP >= 8.3 and Extensions
 
-PHP version and extensions are specified in the `composer.json` file
+The required PHP version and extensions are specified in the composer.json file.
 
+To install PHP 8.3 and the necessary extensions
+1.  Add the repository for PHP 8.3:
 ```bash
-// For MacOX
-brew install php pkg-config imagemagick
-pecl install imagick fileinfo iconv
-// For Linux (depends on the package manger fo course)
-sudo apt update && sudo apt install --no-install-recommends php8.1
-sudo apt-get install -y php8.1-cli php8.1-common php8.1-mysql php8.1-zip php8.1-gd php8.1-mbstring php8.1-curl php8.1-xml php8.1-bcmath php8.1-imagick php8.1-fileinfo php8.1-iconv
+sudo add-apt-repository ppa:ondrej/php
+sudo apt update
+```
+2. Install PHP 8.3 with extensions:
+```bash
+sudo apt install php8.3 php8.3-cli php8.3-{bz2,curl,mbstring,intl}
+sudo apt install php8.3-fpm
+sudo apt-get install -y php8.3-common php8.3-mysql php8.3-zip php8.3-gd php8.3-mbstring php8.3-curl php8.3-xml php8.3-bcmath php8.3-imagick php8.3-fileinfo php8.3-iconv
 pecl install imagick fileinfo iconv
 ```
-
-2) Install Composer globally
-
+3. Install composer globally
 ```bash
 php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
 php -r "if (hash_file('sha384', 'composer-setup.php') === '55ce33d7678c5a611085589f1f3ddf8b3c52d662cd01d4ba75c0ee0459970c2200a51f492d557530c71c15d8dba01eae') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
 php composer-setup.php
 php -r "unlink('composer-setup.php');"
 ```
+4. Install Symfony Binary
 
-3) Install Symfony binary
-
-https://symfony.com/download
-
-
+- For macOS:
 ```bash
-// For MacOX
 brew install symfony-cli/tap/symfony-cli
-// For Linux
-curl -1sLf 'https://dl.cloudsmith.io/public/symfony/stable/setup.deb.sh' | sudo -E bash
-sudo apt install symfony-cli
 ```
-
-
-4) Install MySqL
-
+- For Linux:
 ```bash
-// For MacOX
-brew instal mySql
-// For Linux
 sudo apt install mysql-server
 sudo mysql_secure_installation
 ```
```

Comment:
`sudo apt install mysql-server...` is inside `Install Symfony binaries`

Reply:
And there isn't `Install Mysql` section anymore, oups

Reply:
Modification done.